RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -344,13 +345,14 @@
     qualityCuts = [-0.4,0.0,0.4]
 ))
 
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackdnn.toReplaceWith(pixelLessStep, TrackLwtnnClassifier.clone(
-    src         = 'pixelLessStepTracks',
-    qualityCuts = [-0.6, -0.05, 0.5]
+from RecoTracker.FinalTrackSelectors.TrackTfClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionTf_cfi import *
+trackdnn.toReplaceWith(pixelLessStep, TrackTfClassifier.clone(
+    src = 'pixelLessStepTracks',
+    qualityCuts = qualityCutDictionary['PixelLessStep']
 ))
-(trackdnn & fastSim).toModify(pixelLessStep,vertices = 'firstStepPrimaryVerticesBeforeMixing')
+
```

Comment:
this change with the whitespace and quotes is not strictly needed, please revert